README.md

```diff
@@ -19,15 +51,140 @@ cd acbm
 poetry install
 ```
 
+# How to Run the Pipeline
+
+The pipeline is a series of scripts that are run in sequence to generate the activity-based model. There are a few external datasets that are required. The data and config directories are structured as follows:
+
+```md
+├── config
+│   ├── <your_config_1>.toml
+│   ├── <your_config_2>.toml
+├── data
+│   ├── external
+│   │   ├── boundaries
+│   │   │   ├── MSOA_DEC_2021_EW_NC_v3.geojson
+│   │   │   ├── oa_england.geojson
+│   │   │   ├── study_area_zones.geojson
+│   │   ├── census_2011_rural_urban.csv
+│   │   ├── centroids
+│   │   │   ├── LSOA_Dec_2011_PWC_in_England_and_Wales_2022.csv
+│   │   │   └── Output_Areas_Dec_2011_PWC_2022.csv
+│   │   ├── MSOA_2011_MSOA_2021_Lookup_for_England_and_Wales.csv
+│   │   ├── nts
+│   │   │   ├── filtered
+│   │   │   │   ├── nts_households.parquet
+│   │   │   │   ├── nts_individuals.parquet
+│   │   │   │   └── nts_trips.parquet
+│   │   │   └── UKDA-5340-tab
+│   │   │       ├── 5340_file_information.rtf
+│   │   │       ├── mrdoc
+│   │   │       │   ├── excel
+│   │   │       │   ├── pdf
+│   │   │       │   ├── UKDA
+│   │   │       │   └── ukda_data_dictionaries.zip
+│   │   │       └── tab
+│   │   │           ├── household_eul_2002-2022.tab
+│   │   │           ├── individual_eul_2002-2022.tab
+│   │   │           ├── psu_eul_2002-2022.tab
+│   │   │           ├── trip_eul_2002-2022.tab
+│   │   │           └── <other_nts_tables>.tab
+│   │   ├── travel_times
+│   │   |    ├── oa
+│   │   |    |   ├── travel_time_matrix.parquet
+|   |   |    └── msoa
+│   │   |        └── travel_time_matrix.parquet
+│   │   ├── ODWP01EW_OA.zip
+│   │   ├── ODWP15EW_MSOA_v1.zip
+│   │   ├── spc_output
+│   │   │   ├── <region>>_people_hh.parquet (Generated in Script 1)
+│   │   │   ├── <region>>_people_tu.parquet (Generated in Script 1)
+│   │   │   ├── raw
+│   │   │   │   ├── <region>_households.parquet
+│   │   │   │   ├── <region>_info_per_msoa.json
+│   │   │   │   ├── <region>.pb
+│   │   │   │   ├── <region>_people.parquet
+│   │   │   │   ├── <region>_time_use_diaries.parquet
+│   │   │   │   ├── <region>_venues.parquet
+│   │   │   │   ├── README.md
+│   ├── interim
+│   │   ├── assigning (Generated in Script 3)
+│   │   └── matching (Generated in Script 2)
+│   └── processed
```

Comment:
As discussed, I'll aim to update the notebooks and scripts to read/output files to match this structure.